certdepot/impl_test.go

```diff
@@ -0,0 +1,155 @@
+package certdepot
+
+import (
+	"testing"
+	"time"
+
+	"github.com/square/certstrap/depot"
+	"github.com/stretchr/testify/suite"
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type MongoDepotTestSuite struct {
```

Comment:
what kind of coverage does this provide? are there other tests that we'd want? 

would it be possible to have mongoDepot be of the depot interface? I think in response to my earlier question, we'll have to write another implementation of this that uses the new collection, so we'll want to be able to reuse the test suite on an alternate implementation.

Reply:
yes it should be of the depot interface, that was my bad. and it covers everything we need right now for this interface I think, the functions are fairly straightforward.

Reply:
going to hold off on this, will refactor tests after get this working and in prod